test/functional/eval/msgpack_functions_spec.lua

```diff
@@ -337,7 +337,8 @@ describe('msgpack*() functions', function()
     eq(1, eval('type(parsed[0]) == type(0) ' ..
                '|| parsed[0]._TYPE is v:msgpack_types.integer'))
     if eval('type(parsed[0]) == type(0)') == 1 then
-      eq(1, eval('0xFFFFFFFFFFFFFFFF == parsed[0]'))
+      command('call assert_equal(0xFFFFFFFFFFFFFFFF, parsed[0])')
```

Comment:
It's not entering this branch.  The uint64 test is properly falling into the else.